manifests/integreatly-rhsso/7.5.2-9/rhsso-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,388 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
These files should not be here.
The bundle consists of a manifests folder - CSV and CRDs etc and a metadata folder - with annotations file.

Reply:
Actually no; let's do it same way as it's done for 3scale.
So the actualy files in here are only the manifests; no bundle dockerfile and manifests and annotations folders please.

For reference - this is how it's done for 3scale (or CRO) https://github.com/integr8ly/integreatly-operator/tree/master/manifests/integreatly-3scale/0.9.0

Reply:
@MStokluska  Thank you very much for comment. It was mistake, only in 7.5.2-9, other -18.0 and 7.6.0-4. 
Done. Thanks!